docker/x11vnc.pref

```diff
@@ -0,0 +1,6 @@
+Package: *
```

Comment:
This file to make sure only x11vnc is from cosmic, others should be from bionic.
Ref: https://medium.com/@george.shuklin/how-to-install-packages-from-a-newer-distribution-without-installing-unwanted-6584fa93208f